fiftyone/core/labels.py

```diff
@@ -1448,13 +1448,19 @@ def from_geo_json(cls, d):
 
 
 def _read_mask(mask_path):
-    # pylint: disable=no-member
-    return etai.read(mask_path, flag=cv2.IMREAD_UNCHANGED)
+    if mask_path.endswith(".npy"):
```

Comment:
Not used for the frontend, but we'll run into it when importing / exporting datasets